docs/src/test/java/docs/http/javadsl/server/cors/CorsServerExample.java

```diff
@@ -0,0 +1,65 @@
+package docs.http.javadsl.server.cors;
```

Comment:
all our other `docs` package scala files have license headers - could this be given the standard Apache license header?